department-of-veterans-affairs/va.gov-cms
# VACMS-3607: Disables grid view in Media Library.

## Description

See #3607.

I think something's borked, not sure if it's my dev environment or my changes, so that's why this is a draft request.  I'll update when I get things working again.

## Testing done
Visited the media page before and after the change.  Before,  there 'twas a Grid link.  After, 'twasn't one.

## Screenshots
<img width="771" alt="Screen Shot 2020-12-07 at 1 38 43 PM" src="https://user-images.githubusercontent.com/1318579/101396725-c28b6880-3899-11eb-92d3-60bfaea20b23.png">

## QA steps

As an authenticated user:
1. Visit [`admin/content/media`](http://va-gov-cms.lndo.site/admin/content/media)
1. Confirm there is no `Grid` option.
1. Then validate Acceptance Criteria from issue
- [ ] the Grid view is disabled (not removed) in the Media Library View - /admin/structure/views/view/media_library/edit/page

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
